scripts/just.config.js

```diff
@@ -41,38 +40,35 @@ module.exports = function preset() {
   task('copy', copy);
   task('jest', jest);
   task('jest-watch', jestWatch);
-  task('sass', sass);
   task('ts:commonjs', ts.commonjs);
   task('ts:esm', ts.esm);
-  task('ts:amd', ts.amd);
   task('eslint', eslint);
   task('ts:commonjs-only', ts.commonjsOnly);
   task('webpack', webpack);
-  task('metroPack', metroPackTask(argv()['bundleName']));
+  task('metroPack', () => metroPackTask(argv()['bundleName']));
```

Comment:
what's the difference here?

Reply:
Basically don't run argv directly in the just config, run it when the task is being executed.